src/components/world-map.vue

```diff
@@ -23,6 +23,7 @@ export default class extends Vue {
 
   private map: any | MapEngine = undefined;
   private socketURL: string = 'dvs';
+  private listener: any | Promise<void> = undefined;
```

Comment:
`any | <other type>` is equivalent to `any`. If you want to express nullability, you may use
`private listener?: Promise<void> = undefined;`

Reply:
should we also chage `private map: any | MapEngine = undefined;`?

Reply:
to be honest, yes